fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -173,6 +177,14 @@ protected void auditAfterExec(String origStmt, StatementBase parsedStmt,
 
     // only throw an exception when there is a problem interacting with the requesting client
     protected void handleQuery(MysqlCommand mysqlCommand, String originStmt) {
+        try {
+            executeQuery(mysqlCommand, originStmt);
+        } catch (Exception ignored) {
+            // saved use handleQueryException
```

Comment:
why ignore the exception

Reply:
because exception saved use handleQueryException, same as before.